README.md

```diff
@@ -0,0 +1,95 @@
+# Serverless - AWS Node.js Typescript
```

Comment:
Hi Diego, where I can find link to deployed functions?

Have you added any swagger documentation?

Reply:
Hi @pkhlebko, apologies for not including the functions endpoints and contracts, I've added the info in the PR description.

Reply:
I've also updated the README file with more detailed information (: